From 5880f146c24a582decff89b2980d27b78a9c8058 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C4=90=E1=BB=97=20Ho=C3=A0ng?= <dohoho.23.95@gmail.com>
Date: Mon, 27 Aug 2018 10:18:18 +0700
Subject: [PATCH 1/2] add header file

---
 hoclinux.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hoclinux.cc b/hoclinux.cc
index e69de29..bfcdbf0 100644
--- a/hoclinux.cc
+++ b/hoclinux.cc
@@ -0,0 +1 @@
+#include <stdio.h>
-- 
2.7.4

